Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.h

```diff
@@ -47,7 +47,7 @@ namespace EMotionFX
         ColliderPropertyNotify(ColliderWidget* colliderWidget);
 
         void BeforePropertyModified(AzToolsFramework::InstanceDataNode* pNode) override;
-        void AfterPropertyModified(AzToolsFramework::InstanceDataNode* /*pNode*/) override {}
+        void AfterPropertyModified(AzToolsFramework::InstanceDataNode* pNode) override;
 
         void SetPropertyEditingActive(AzToolsFramework::InstanceDataNode* /*pNode*/) override {}
```

Comment:
nit: Might as well update this while we're here too

Reply:
Done.